app/web/src/components/Customizer/customizerConverter.ts

```diff
@@ -0,0 +1,120 @@
+// CadHub
+
+type CadhubTypeNames = 'number' | 'string' | 'boolean'
+
+interface CadhubParamBase {
+  type: CadhubTypeNames
+  caption: string
+  name: string
+}
+export interface CadhubStringParam extends CadhubParamBase {
+  type: 'string'
+  initial: string
+  placeholder?: string
+  maxLength?: number
+}
+export interface CadhubBooleanParam extends CadhubParamBase {
+  type: 'boolean'
+  initial?: boolean
+}
+export interface CadhubNumberParam extends CadhubParamBase {
+  type: 'number'
+  initial: number
+  min?: number
+  max?: number
+  step?: number
+  decimal?: number
+}
+
+export type CadhubParams =
+  | CadhubStringParam
+  | CadhubBooleanParam
+  | CadhubNumberParam
+
+// OpenSCAD
+const openscadValues = `
```

Comment:
The following is just here to test implementation for OpenSCAD and will be removed later.